service/controller/resource/spark/create.go

```diff
@@ -22,26 +26,37 @@ import (
 	expcapiv1alpha3 "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/giantswarm/azure-operator/v4/pkg/label"
 	"github.com/giantswarm/azure-operator/v4/service/controller/cloudconfig"
 	"github.com/giantswarm/azure-operator/v4/service/controller/encrypter"
 	"github.com/giantswarm/azure-operator/v4/service/controller/key"
 )
 
 const (
-	ignitionBlobKey = "ignitionBlob"
+	dockerVolumeSizeGB  = 50
```

Comment:
Can't we un-hardcode these values? It would be nice to be able to set them on a per-cluster basis

Reply:
I don't think these values are even used.

We just need a valid `AzureConfig` object that renders the required cloudconfig ignition files. This part of the code has nothing to do with Azure disks.

Reply:
Maybe I missed it @whites11 but I couldn't find anywhere in `k8scloudconfig`where these values are used.